src/style.less

```diff
@@ -13,6 +13,7 @@
 	.rootContainer {
 		border: 1px solid black;
 		padding: 20px;
+		margin-left: 105px;
```

Comment:
This visualization is for use in Bluegenes, so the extra margin would add a lot of empty space when the viz is displayed inside Bluegenes. I agree that it makes it nicer to look at when running `npm run dev`, but a nicer displayer for each of the tools is something we want to work on in the future.

Sorry that this hasn't been specified properly!

To summarise, please remove the margin-left property.